products/sle/main.pm

```diff
@@ -333,6 +333,12 @@ if (is_updates_test_repo && !get_var('MAINT_TEST_REPO')) {
     set_var('ADDONURL',     '');
     set_var('ADDONURL_SDK', '');
 
+    # SLES4SAP does not have addon on SLE12SP3, is not present in SCC_ADDONS
+    if (is_sle('<15') && check_var('FLAVOR', 'Server-DVD-SLES4SAP-Updates')) {
+        $incidents{SLES4SAP} = get_var(uc($a) . '_TEST_ISSUES');
```

Comment:
`uc($a) `are you sure I think right is here `get_var('SLE4SAP_TEST_ISSUES','')`

or you can push `sle4sap` to `@addons` --> `push @addons 'sle4sap'`

Reply:
no i can't, then it will look for SLES4SAP module

Reply:
fixed